src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -89,6 +80,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return 0;
     }
 
+    /**
+     * @return array<int, mixed[]>
```

Comment:
This does not add much over the `: array` return type. In general, Symfony favors less PHPdoc.